meta/main.yml

```diff
@@ -5,19 +5,12 @@ galaxy_info:
   company: Red Hat, Inc.
   license: MIT
   min_ansible_version: "2.9"
-  platforms:
```

Comment:
Why do you remove this? We need this for ci and generally for role best practices.

Reply:
I tried to find out, what to enter for sles/opensuse and found https://github.com/ansible/ansible/issues/82453 which states, that this is not used anymore by the ansible galaxy.

Reply:
Our CI uses it to determine for what platforms to run tests.

Reply:
what should I put in there for suse? Or just leave suse out?

Reply:
Let's put SUSE according to https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/facts/system/distribution.py#L68
```
    - name: SUSE
      versions:
        - "15"
```

Reply:
Sorry, `SUSE` doesn't pass ansible-lint. And ansible-lint knows better, please use `opensuse` as per https://github.com/ansible/ansible-lint/blob/main/src/ansiblelint/schemas/meta.json#L1269